src/envoy/mixer/http_filter.cc

```diff
@@ -242,6 +242,8 @@ class Instance : public Http::StreamFilter, public Http::AccessLog::Instance {
                    const HeaderMap* response_headers,
                    const AccessLog::RequestInfo& request_info) override {
     Log().debug("Called Mixer::Instance : {}", __func__);
+    // If decodeHaeders() is not called, not to call Mixer report.
+    if (!request_data_) return;
```

Comment:
aah, ok. So if there is a connection reset or other transport level error, there will not be any headers to decode, but `Log` will still be called.